bin/boot

```diff
@@ -25,6 +25,36 @@ function login  {
 }
 
 function start {
+    if [ "$ADMIN_PASSWORD" ]; then
+        sed -i "s/_ADMIN_PASSWORD_/$ADMIN_PASSWORD/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$DB_USER" ]; then
+        sed -i "s/_DB_USER_/$DB_USER/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$DB_PASSWORD" ]; then
+        sed -i "s/_DB_PASSWORD_/$DB_PASSWORD/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$ADDONS_REPO" ]; then
+        echo "Setting up SSH key"
+        cp -R /mnt/ssh /root/.ssh
+        chown -R $USER:$USER /root/.ssh
+        ssh_file="/root/.ssh/id_rsa"
```

Comment:
Maybe it would be better to have the filename of the SSH key as an ENV variable? Also, I'm wondering if for some cases we could need more than 1 SSH key (for instance if we pull the code from several repos that don't have the same deployment key). Maybe simply:
`chmod 600 /root/.ssh/*`